.travis.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'English'
+require 'benchmark'
+
+def jruby?
+  RUBY_PLATFORM == 'java'
```

Comment:
I think that `RUBY_ENGINE == 'jruby'` is more intentional. Also I saw it with existing code.

e.g. https://github.com/bbatsov/rubocop/blob/v0.49.1/spec/support/jruby_workaround.rb#L6